app/linux/src/LightingManager.cpp

```diff
@@ -45,34 +47,41 @@ CHIP_ERROR LightingManager::Init()
     char *envGPIO = std::getenv(GPIO);
     if (envGPIO == NULL || strlen(envGPIO) == 0)
     {
-        ChipLogError(AppServer, "Environment variable not set or empty: %s", GPIO);
-
+        ChipLogError(AppServer, "Unset or empty environment variable: %s", GPIO);
         return CHIP_ERROR_INVALID_ARGUMENT;
     }
-
     ChipLogProgress(AppServer, "Using GPIO %s", envGPIO);
 
-    gpio = std::stoi(envGPIO);
+    char *envGPIOCHIP = std::getenv(GPIOCHIP);
+    if (envGPIOCHIP == NULL || strlen(envGPIOCHIP) == 0)
+    {
+        ChipLogError(AppServer, "Unset or empty environment variable: %s", envGPIOCHIP);
```

Comment:
Here it should be `GPIOCHIP` instead of `envGPIOCHIP` to be coherent with the previous in line `50`.

Reply:
I'll fix it latter!